user_data/integration_test_config.yml

```diff
@@ -0,0 +1,9 @@
+# Parameters for running the integration test script
+# A reasonably large, well-resolved box, with otherwise completely
+# default parameters. If defaults change, this will *break* the integration
+# test, but this is almost always what we want (we don't want defaults to change
+# without our express permission).
+user_params:
+  BOX_LEN: 400.0
+  HII_DIM: 400
```

Comment:
This setup could take quite some time to run, and be quite intensive on memory resources (i.e. will not be able to be run on laptops or desktop PCs). Not sure how rigid the plan is to be with these.

Reply:
This file is not actually used at all... I've now deleted it. The HII_DIM of the automatic unit-tests is 50, while the more involved PR feature plots use 128. Does this seem reasonable?